.gitignore

```diff
@@ -0,0 +1,15 @@
+# Java build artifacts class files
```

Comment:
Please provide a `.gitignore` that is the same as in the other fixtures. See https://github.com/test-editor/rest-fixture/blob/develop/.gitignore